Assets/Scripts/EOSSessionsManager.cs

```diff
@@ -331,6 +345,81 @@ public override int GetHashCode()
         {
             return base.GetHashCode();
         }
+
+        /// <summary>
+        /// Determines if the local user owns this Session.
+        /// This information can be used to gate some Session modifying actions.
+        /// </summary>
+        /// <returns>
+        /// True if the Session is owned by the local user.
+        /// False if either it was determined that the user doesn't own this Session,
+        /// or if the operation to query that information failed.
+        /// </returns>
+        public bool IsLocalUserOwnerOfSession
+        {
+            get
+            {
+                ProductUserId localUserId = EOSManager.Instance?.GetProductUserId();
+
+                // If we can't even get the local user's id, that state suggests they shouldn't be owning any Sessions
+                if (localUserId == null)
+                {
+                    return false;
+                }
+
+                // If the ActiveSession isn't set, then this user isn't a part of the Session
+                // Can't possibly own it
+                if (ActiveSession == null)
+                {
+                    return false;
+                }
+
+                ActiveSessionCopyInfoOptions options = new ActiveSessionCopyInfoOptions() { };
+                Result copyResult = ActiveSession.CopyInfo(ref options, out ActiveSessionInfo? sessionInfo);
+
+                if (copyResult != Result.Success || !sessionInfo.HasValue || !sessionInfo.Value.SessionDetails.HasValue)
+                {
+                    return false;
+                }
+
+                return sessionInfo.Value.SessionDetails.Value.OwnerUserId.Equals(localUserId);
+            }
+        }
+
+        public bool TryGetRegisteredUsers(out List<ProductUserId> userIds)
+        {
+            if (ActiveSession == null)
+            {
+                userIds = null;
+                return false;
+            }
+
+            ActiveSessionCopyInfoOptions copyInfoOptions = new ActiveSessionCopyInfoOptions() { };
+            ActiveSessionInfo? copiedInfo;
+            Result activeSessionInfoCopyResult = ActiveSession.CopyInfo(ref copyInfoOptions, out copiedInfo);
+
+            if (activeSessionInfoCopyResult != Result.Success)
+            {
+                Debug.LogError($"{nameof(EOSSessionsManager)} ({nameof(TryGetRegisteredUsers)}): Failed to copy session information. Result code {activeSessionInfoCopyResult}.");
+                userIds = null;
+                return false;
+            }
+
+            ActiveSessionGetRegisteredPlayerCountOptions countOptions = new ActiveSessionGetRegisteredPlayerCountOptions() { };
```

Comment:
I think it is acceptable - nay preferable to (particularly when the type name is long) use the simple `new()` means. In this particular case, you can also omit the curly brackets at the end. This may be a preference thing (the use of `new()`, not the curly brackets - either way it makes sense to remove those).

Reply:
Honestly most of the time we have these { }s after these structs, it's to see if anything should go in there. The answer for this one was no, there's nothing inside.

Mint is not fully on board with the new() syntax, but at least in this case the name of the object is in the variable declaration, so it's fine here.